### PR TITLE
[REPO SYNC] molecule v2, circleci, better repo creation, update docs

### PR DESCRIPTION
**Pull request by SuperQ**
One or more files which should be in sync across cloudalchemy repos were changed either here or in [cloudalchemy/skeleton](https://github.com/cloudalchemy/skeleton).
This PR propagates files from [cloudalchemy/skeleton](https://github.com/cloudalchemy/skeleton). If something was changed here, please first modify skeleton repository.

CC: @paulfantom.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -34,17 +34,17 @@ Some great guidelines can be found [here](https://wiki.openstack.org/wiki/GitCom
 ## Releases
 
 We try to stick to semantic versioning and our releases are automated. Release is created by assigning a keyword (in a
-way similar to travis [`[ci skip]`](https://docs.travis-ci.com/user/customizing-the-build#Skipping-a-build)) to a 
-commit with merge request. Available keywords are (square brackets are important!):
+way similar to circle ci keyword [`[ci skip]`](https://docs.travis-ci.com/user/customizing-the-build#Skipping-a-build))
```
Comment (paulfantom) [inline CONTRIBUTING.md:37]: nit: link is incorrect

Comment (paulfantom) [inline CONTRIBUTING.md:37]: nit: we should probably link to circle ci docs

Comment (SuperQ) [reply CONTRIBUTING.md:37]: Yea, things to fix in the skeleton repo :)

**Comment by SuperQ** [comment]
FYI, I'm just testing out the sync_skeleton script here. Do not merge.

**Review by paulfantom — APPROVED** [review]
lgtm, just one nit that can be fixed later

**Pull request closed without merging**